internal/csi-addons/rbd/replicationcontrollerserver.go

```diff
@@ -683,7 +637,7 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	err = repairResyncedImageID(ctx, rbdVol, ready)
+	err = rbdutil.RepairResyncedImageID(ctx, rbdVol, ready)
```

Comment:
Same here, it would be cleaner to have `RepairResyncedImageID` a function of `rbdVol` so you can do

```go
err = rbdVol.RepairResyncedImageID(ctx, ready)
```